backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveDiskCommand.java

```diff
@@ -170,11 +170,13 @@ protected Map<String, Pair<String, String>> getExclusiveLocks() {
 
     private String getDiskIsBeingMigratedMessage() {
         if (cachedDiskIsBeingMigratedMessage == null) {
+            DiskImage diskImage = diskImageDao.getAncestor(getParameters().getImageId());
+            String diskName = getParameters().getNewAlias() != null ? getParameters().getNewAlias() :
+                    diskImage.getDiskAlias() != null ?  diskImage.getDiskAlias() : "";
```

Comment:
I see two problems with this code:
1. diskImage might be null and in that case we'll get NPE when invoking `diskImage.getDiskAlias()`
2. we retrieve the diskImage from the database also when `getParameters().getNewAlias() != null`, in which case we don't need the result from the database